wildfly/amq-broker-ssl/src/main/java/org/jboss/intersmash/applications/amq/broker/ssl/JmsTestConstants.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.intersmash.applications.amq.broker.ssl;
+
+/**
+ * Constants used for JMS testing with ActiveMQ message broker.
+ * <p>
+ * This class defines message content and response templates used by the JMS test servlet
+ * and message-driven beans during testing of message broker interactions.
+ * </p>
+ */
+public class JmsTestConstants {
+
+	/**
+	 * Response message template when a message is successfully sent to a queue.
+	 * The actual queue name will be appended to this string.
+	 */
+	static final String QUEUE_SEND_RESPONSE = "Sent a text message to ";
+
+	/**
+	 * Default text content for test messages sent from the servlet to a queue.
+	 */
+	static final String QUEUE_TEXT_MESSAGE = "Hello Servlet!";
+
+	/**
+	 * Response message template when a message is successfully sent to the MDB queue.
+	 * The actual queue name will be appended to this string.
+	 */
+	static final String QUEUE_MDB_SEND_RESPONSE = "Sent a text message with to ";
```

Comment:
Maybe we can consider whether we can fix this constant now, e.g.: s/with/(MDB)/ or something like that? WDYT?